src/patterns.rs

```diff
@@ -362,19 +362,31 @@ fn rewrite_tuple_pat(
     let path_str = path_str.unwrap_or_default();
     let pat_ref_vec = pat_vec.iter().collect::<Vec<_>>();
 
-    overflow::rewrite_with_parens(
-        &context,
-        &path_str,
-        &pat_ref_vec,
-        shape,
-        span,
-        context.config.max_width(),
-        if add_comma {
-            Some(SeparatorTactic::Always)
-        } else {
-            None
-        },
-    )
+    if !dotdot_pos.is_none() {
+        overflow::rewrite_with_parens(
+            &context,
+            &path_str,
+            &pat_ref_vec,
+            shape,
+            span,
+            context.config.max_width(),
+            Some(SeparatorTactic::Never),
+        )
+    } else {
+        overflow::rewrite_with_parens(
+            &context,
+            &path_str,
+            &pat_ref_vec,
+            shape,
+            span,
+            context.config.max_width(),
+            if add_comma {
```

Comment:
Could move the `is_some` check to here to prevent duplicating the other parameters.